install/tools/setupMeshEx.sh

```diff
@@ -51,37 +51,25 @@ function istioDnsmasq() {
   for _ in {1..20}; do
     PILOT_IP=$(kubectl get -n "$NS" service istio-pilot-ilb -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
     ISTIO_DNS=$(kubectl get -n kube-system service dns-ilb -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    MIXER_IP=$(kubectl get -n "$NS" service mixer-ilb -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    CITADEL_IP=$(kubectl get -n "$NS" service citadel-ilb -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 
-    if [ "${PILOT_IP}" == "" ] || [  "${ISTIO_DNS}" == "" ] || [ "${MIXER_IP}" == "" ] || [ "${CITADEL_IP}" == "" ] ; then
-      echo "Waiting for ILBs, pilot=$PILOT_IP, MIXER_IP=$MIXER_IP, CITADEL_IP=$CITADEL_IP, DNS=$ISTIO_DNS - kubectl get -n $NS service: $(kubectl get -n "$NS" service)"
+    if [ "${PILOT_IP}" == "" ] || [  "${ISTIO_DNS}" == "" ] ] ; then
```

Comment:
```suggestion
    if [ "${PILOT_IP}" == "" ] || [  "${ISTIO_DNS}" == "" ] ; then
```

Reply:
Got it, linter complained about all of them. 
The real problem is that the script is not yet run in the test (and the second problem is that the test is not called by the CI/CD)